refactor(cli): adding prompts for pk key and type for relation cli (#6489)
strongloop/loopback-next: docs/site/Relation-generator.md

- `<destinationModel>` - Name of the model to create a relationship with.

- `<sourceModelPrimaryKey>` - Name of the id property on the model to create the
  relationship from.

> - `<sourceModelPrimaryKeyType>` - Type of the id property on the source mode.

Reviewer: ```suggestion
- `<sourceModelPrimaryKeyType>` - Type of the id property on the source model.
```

- `<destinationModelPrimaryKey>` - Name of the id property on the model to
  create a relationship with.

- `<destinationModelPrimaryKeyType>` - Type of the id property on the

>   destination mode.

Reviewer: ```suggestion
  destination model.
```

lb4 relation --sourceModel=<sourceModel>
--destinationModel=<destinationModel> --throughModel=<throughModel>
--relationType=<hasManyThrough> [--relationName=<relationName>]

> [--sourceModelPrimaryKey=<sourceModelPrimaryKey>]

Reviewer: For `HasManyThrough`, is `sourceModelPrimaryKey` the same as `sourceKeyOnThrough` and `destinationModelPrimaryKey` the same as `targetKeyOnThrough`?

Author: Good question! Indeed they refer to the same thing, but they may be different. For example, `sourceModelPrimaryKey` on `Customer` is `id`, and `sourceKeyOnThrough` on `CustomerOrderLink` defaults to `customerId`, and `targetKeyOnThrough` defaults to `orderId`. Therefore those two params are needed IMO.

  to choose from as the through model of the relation. For HasManyThrough
  relation only.

- Name of the id property of the source model (`sourceModelPrimaryKey`). Prompts
  for the name of the source model. For example, for `Customer` has many
  `Order`s, `Customer` is the source model. For an `Order` belongs to a
  `Customer`, `Order` is the source model. Defaults value: `id`.

> - Type of the id property of the source model (`sourceModelPrimaryKey`). Prompts

Reviewer: ```suggestion
- Type of the id property of the source model (`sourceModelPrimaryKeyType`). Prompts
```

>   a list of types fir the id property on the source model. Default value:

Reviewer: ```suggestion
  a list of types for the id property on the source model. Default value:
```

  `number`.

> - Name of the id property of the source model (`destinationModelPrimaryKey`).

Reviewer: ```suggestion
- Name of the id property of the destination model (`destinationModelPrimaryKey`).
```

  Prompts for the name of the destination model. For example, for `Customer` has
  many `Order`s, `Order` is the destination model. For an `Order` belongs to a

>   `Customer`, `Customer` is the destination model. Defaults value: `id`.

Reviewer: ```suggestion
  `Customer`, `Customer` is the destination model. Default value: `id`.
```

> - Type of the id property of the source model (`sourceModelPrimaryKey`). Prompts
>   a list of types fir the id property on the source model. Default value:
>   `number`.
> 

Reviewer: ```suggestion
- Type of the id property of the destination model (`destinationModelPrimaryKeyType`). Prompts
  a list of types for the id property on the source model. Default value:
  `number`.
```
